README.Rmd

```diff
@@ -15,153 +17,126 @@ knitr::opts_chunk$set(
 )
 ```
 
-# Rewrite of [r-polars](https://github.com/pola-rs/r-polars)
+# polars _(R Polars)_
 
-## Motivation
+<!-- TODO: add link to discord -->
 
-I have been developing r-polars for over a year, and I felt that a significant rewrite was necessary.
-r-polars is a clone of [py-polars](https://github.com/pola-rs/polars/tree/main/py-polars) /
-[polars-python](https://github.com/pola-rs/polars/tree/main/crates/polars-python),
-but the package structure is currently quite different.
-Therefore, it was difficult to keep up with frequent updates.
+<!-- badges: start -->
+[![R-multiverse status](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2Fpolars&query=%24.Version&label=r-multiverse)](https://community.r-multiverse.org/polars)
+[![R-universe status badge](https://rpolars.r-universe.dev/badges/polars)](https://rpolars.r-universe.dev)
+[![CRAN status](https://www.r-pkg.org/badges/version/polars)](https://CRAN.R-project.org/package=polars)
+[![Docs dev version](https://img.shields.io/badge/docs-dev-blue.svg)](https://pola-rs.github.io/r-polars)
+<!-- badges: end -->
 
-I thought that now, around the release of Python Polars 1.0.0, is a good time for a complete rewrite, so I decided to try it.
+Polars is a blazingly fast DataFrame library for manipulating structured data.
+The core is written in Rust.
 
-There are several reasons to rewrite r-polars on both the Rust and R sides.
+The goal of Polars is to provide a lightning fast DataFrame library that:
 
-### Rust side
+- Utilizes all available cores on your machine.
+- Optimizes queries to reduce unneeded work/memory allocations.
+- Handles datasets much larger than your available RAM.
+- A consistent and predictable API.
+- Adheres to a strict schema (data-types should be known before running the query).
 
-1. Appropriate file division. Due to the limitations of [extendr](https://github.com/extendr/extendr),
-   it is not possible to place multiple impl blocks.
-   ([extendr/extendr#538](https://github.com/extendr/extendr/issues/538))
-2. Error handling.
-   There is a lot of custom code to use the Result type with extendr, which is quite different from other packages based on extendr.
-   ([extendr/extendr#650](https://github.com/extendr/extendr/issues/650))
-3. Simplify type conversion.
-   The code is difficult to follow because it uses a macro called `robj_to` for type conversion (at least in rust-analyzer).
+This `polars` R package provides the R bindings for Polars.
+It can be used to convert R DataFrames to Polars DataFrames and vice versa,
+as well as to integrate with other common R packages.
```

Comment:
I feel it's a bit reductive to mention only conversion between R and Polars. How about this?

```suggestion
It can be used to process Polars DataFrames and LazyFrames, convert objects 
between Polars and R, and can be integrated with other common R packages.
```

Reply:
Again, I'm not sure if the term LazyFrame is appropriate here; I believe LazyFrame is commonly referred to as a Query Plan or something similar.

Reply:
How about 54ad963d1502264065b3d13a8cabddf5eddd2623?

Reply:
It appears many times in the user guide, even when introducing the lazy api: https://docs.pola.rs/user-guide/lazy/

There's even a function called LazyFrame so I don't see a problem with that.

Reply:
> How about [54ad963](https://github.com/eitsupi/neo-r-polars/commit/54ad963d1502264065b3d13a8cabddf5eddd2623)?

Fine for  me, I just don't want to replace all "LazyFrame" in docs by query plans.